src/main/java/model/RandomGenerator.java

```diff
@@ -0,0 +1,33 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Collections.shuffle;
+import static java.util.List.*;
+
+public class RandomGenerator {
+    public final static int[] RANDOM_NUM_LIST = new int[10];
+    private final static List<Integer> NUM_LIST = of(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+
+    private static int makeRanNum() {
+        List<Integer> copy = new ArrayList<>(NUM_LIST);
+        shuffle(copy);
+
+        for (int i=0; i<10; i++) {
+            RANDOM_NUM_LIST[i] = copy.get(i);
+        }
+
+        return RANDOM_NUM_LIST[0];
```

Comment:
```suggestion
        return copy.get(0);
```
모든 값을 저장하지 않고 첫번쨰 요소만 리턴하는것도 가능할것 같네요 😄

Reply:
해당 방법이 있었네요...
Random으로 전체 변경했습니다!

[f2af969](https://github.com/next-step/java-racingcar-kakao/pull/16/commits/f2af969b964cbc0750f195ce30b942d028b44623)